smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/readwrite/DocumentReadingClosureUtils.kt

```diff
@@ -1,68 +1,27 @@
 package software.amazon.smithy.swift.codegen.integration.serde.readwrite
 
-import software.amazon.smithy.codegen.core.Symbol
 import software.amazon.smithy.model.shapes.MemberShape
-import software.amazon.smithy.model.shapes.Shape
-import software.amazon.smithy.swift.codegen.ClientRuntimeTypes
-import software.amazon.smithy.swift.codegen.SmithyXMLTypes
-import software.amazon.smithy.swift.codegen.SwiftDependency
 import software.amazon.smithy.swift.codegen.SwiftWriter
 import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
-import software.amazon.smithy.swift.codegen.integration.serde.xml.NodeInfoUtils
 
 class DocumentReadingClosureUtils(
```

Comment:
Updated reading closure generation for closure-based serde in all three protocols.